package.json

```diff
@@ -25,20 +25,21 @@
     "url": "https://github.com/Pomax/react-onclickoutside/issues"
   },
   "scripts": {
+    "build": "rollup -c -i src/index.js -o ./index.js",
     "test": "npm run lint && karma start test/karma.conf.js --single-run && npm run test:nodom",
     "test:nodom": "mocha test/no-dom-test.js",
-    "lint": "eslint index.js ./test",
-    "precommit": "npm run lint"
-  },
-  "dependencies": {
-    "create-react-class": "^15.5.x"
+    "lint": "eslint src/*.js ./test",
+    "precommit": "npm run build && npm run test"
```

Comment:
we want "npm run build" to be part of the "test" script, so that we're always testing against the result of a build, rather than only doing so during precommit.